src/Symfony/Component/Notifier/Event/NotificationEvents.php

```diff
@@ -43,7 +43,7 @@ public function getEvents(string $name = null): array
 
         $events = [];
         foreach ($this->events as $event) {
-            if ($name === $event->getMessage()->getTransport()) {
+            if ($name === (string) $event->getMessage()->getTransport()) {
```

Comment:
not sure about this one: do we support the empty string as transport name?

Reply:
Yes